PerDiem/PerDiem/TransactionCell.m

```diff
@@ -14,6 +25,27 @@ - (void)awakeFromNib {
     // Initialization code
 }
 
+- (void)setTransaction:(Transaction *)transaction {
+    _transaction = transaction;
+
+    self.dateLabel.text = [self formatDate:self.transaction.transactionDate];
+    self.budgetLabel.text = self.transaction.budget.name;
+    self.paymentTypeLabel.text = self.transaction.paymentType.name;
+    self.summaryLabel.text = self.transaction.summary;
+
+    // @TODO - make common currency formatter method to be used throughout app.
```

Comment:
here's an example on how to create a currency formatter:

in private interface:
```objective-c
@property (strong, nonatomic) NSNumberFormatter *amountFormatter;
```

in implementation:
```objective-c
- (NSNumberFormatter *)amountFormatter {
    if (!_amountFormatter) {
        _amountFormatter = [[NSNumberFormatter alloc] init];
        [_amountFormatter setNumberStyle: NSNumberFormatterCurrencyStyle];
    }
    return _amountFormatter;
}
```